packages/calcite-design-tokens/.npmignore

```diff
@@ -0,0 +1,33 @@
+# Logs
+logs
+*.log
+
+# Dependency directory
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
+node_modules
+
+# macOS metadata
+.DS_Store
+
+# Pre-build Folders
+src
+support
+
+# token-transformer runner
+bin
+
+# Documentation
+CONTRIBUTING.md
+
+# Configurations
+.editorconfig
+.eslintrc.json
+.gitignore
+.github
+.husky
+.vscode
+.lintstagedrc.json
+.prettierignore
+.prettierrc
+jest.config.json
+tsconfig*
```

Comment:
I'd still prefer if we used the `files` property in `package.json` instead of `.npmignore`. But if you're set on `.npmignore`, can you remove all of the patterns that aren't relevant to this package.